### ISSUE TITLE
working with binary dependent variable

### ISSUE THREAD

**Issue by SunainaD** [sergiocorreia/reghdfe#179]
Hi, Is it possible to use reghdfe command to absorb fixed effects for a binary outcome variable?

**Comment by sergiocorreia** [comment]
Yes, but as a linear probability model, not as logit/probit (for that you would need to do it within a GLM-type command)

**Comment by SunainaD** [comment]
Thanks! also, the results with reghdfe and xtreg, fe for linear model differs. My understanding is that the xtreg takes into account the panel nature/setting of the data whereas as reghdfe, like areg, only hides the additional dummies. Please correct me.

**Comment by sergiocorreia** [comment]
Reghdfe can work as xtreg and areg depending on what you want.

When you say results differ, what exactly is differing? Point estimates or SEs?

**Comment by SunainaD** [comment]
My bad, i should have mentioned that. the standard errors are larger with the xtreg, fe. I actually want to use clustered standard errors xtreg, fe doesn’t allow me to cluster at a level nested within the panel id so I just tried with the robust option.

**Comment by JakeNaismith** [comment]
Hello sergiocorreia,

Can i run a linear probability model using reghdfe? My command is as follows:
`reghdfe Y l.X, a(A B) vce(robust)`

Is that correct? Thank you

**Comment by JakeNaismith** [comment]
Hello sergiocorreia,

Can i run a linear probability model using reghdfe? My command is as follows:

`reghdfe Y l.X, a(A B) vce(robust)`
Is that correct? Thank you
